sympy/solvers/ode/systems.py

```diff
@@ -160,7 +174,35 @@ def linodesolve_type(A, t, b=None):
                 by linodesolve.
             '''))
 
-    return {"type": type, "antiderivative": B}
+        match['antiderivative'] = B
+        match.update(_first_order_type5_6_subs(A, t, b=b))
+
+    return match
+
+
+def _first_order_type5_6_subs(A, t, b=None):
+    match = {}
+
+    factor_terms = _factor_matrix(A, t)
+    is_homogeneous = b is None or b.is_zero_matrix
+
+    if factor_terms is not None:
+        t_ = Symbol("{}_".format(t))
+        F_t = integrate(factor_terms[0], t)
+        inverse = solveset(Eq(t_, F_t), t)
+
+        # Note: A simple way to check if a function is invertible
```

Comment:
@oscarbenjamin Does this method seem correct?

Reply:
Yes, although ideally there should be a function for computing the inverse of a sympy expression.

Reply:
I was also thinking of filtering inverses which has `Piecewise` operations on it to avoid overcomplicating the solutions when it doesn't have to be.